design/ceph/ceph-telemetry.md

```diff
@@ -0,0 +1,77 @@
+# Adding Rook data to Ceph telemetry
+The solution plan agreed upon with the telemetry team is for the Rook operator to add telemetry to
+the Ceph mon `config-key` database, and Ceph will read each of those items for telemetry retrieval.
+See: https://github.com/rook/rook/pull/10161
+
+## Guidelines from the Ceph telemetry team
+- Users must opt in to telemetry
+- Users must also opt into the new (delta) telemetry items added between Ceph versions
+- Rook should not add information to `config-key` keys that can grow arbitrarily large to keep space
+  usage of the mon database low.
+
+## Proposed metrics for Rook to collect
+- `rook-version` - Rook version.
+  - Done in https://github.com/rook/rook/pull/10161
```

Comment:
Instead of referring to PRs when each one is implemented, how about if we put all the telemetry name consts in a single place (perhaps create a telemetry.go), and link it from this doc?